src/test/java/org/opensearch/sdk/TestExtensionsRunner.java

```diff
@@ -102,7 +103,7 @@ public void testStartTransportService() {
         verify(transportService, times(5)).registerRequestHandler(anyString(), anyString(), anyBoolean(), anyBoolean(), any(), any());
     }
 
-    @Test
+    // @Test
```

Comment:
Is there a reason this is commented out?

Reply:
oops sorry @ryanbogan , I thought it was commented before and I had uncommented it locally to test. Looks like I was wrong. I will revert back this change

Thanks and Regards
Varun

Reply:
Hi @ryanbogan,

Actually this PR is stale. I had created a new PR as this was old and had some 15+ commits behind from main and was giving lot of merge conflicts . To save the pain of rebasing it, I had created a altogether fresh PR here. https://github.com/opensearch-project/opensearch-sdk-java/pull/508. Can you please review this ?

I will delete this PR if its okay.

Thanks and Regards
Varun

Reply:
Sure! I'll take a look now